src/ReactiveUI.Tests/ReactiveCollectionTest.cs

```diff
@@ -984,6 +984,82 @@ public void DerviedCollectionShouldHandleItemsRemoved()
             Assert.Equal(disposed.Count, 2 + count);
         }
 
+        [WpfFact]
+        public void DataboundReactiveListDoesNotThrowForAddRange()
+        {
+            var vm = new PropertyBindViewModel();
+            var view = new PropertyBindView { ViewModel = vm };
+            var fixture = new PropertyBinderImplementation();
+            fixture.OneWayBind(vm, view, m => m.SomeCollectionOfStrings, v => v.FakeItemsControl.ItemsSource);
+            // eliminate the ResetChangeThreshold from the equation
+            vm.SomeCollectionOfStrings.ResetChangeThreshold = int.MinValue;
+
+            var exception = Record.Exception(() => {
+
+                // Within the reset threshold
+                vm.SomeCollectionOfStrings.AddRange(Create(5));
+                vm.SomeCollectionOfStrings.AddRange(Create(20));
+            });
+
+            Assert.Null(exception);
+
+            IEnumerable<string> Create(int numElements)
+                => Enumerable.Range(1, numElements).Select(i => $"item_{i}");
+        }
+
+        [WpfFact]
+        public void DataboundReactiveListDoesNotThrowForInsertRange()
+        {
+            var vm = new PropertyBindViewModel();
+            var view = new PropertyBindView { ViewModel = vm };
+            var fixture = new PropertyBinderImplementation();
+            fixture.OneWayBind(vm, view, m => m.SomeCollectionOfStrings, v => v.FakeItemsControl.ItemsSource);
+            vm.SomeCollectionOfStrings.ResetChangeThreshold = int.MinValue;
+
+            foreach (var item in Create(5))
+            {
+                vm.SomeCollectionOfStrings.Add(item);
+            }
+
+            var exception = Record.Exception(() => {
```

Comment:
Given our expectation is no exception, do we gain anything by trapping and asserting later (wouldn't the test just throw anyway?)